xontrib/cmd_done.py

```diff
@@ -9,13 +9,12 @@
 TRIGGER_NOTIFICATION = xsh.env.get("XONTRIB_CD_TRIGGER_NOTIFICATION", True)
 NOTIFICATION_APP_NAME = xsh.env.get("XONTRIB_CD_NOTIFICATION_APP_NAME", xsh.env.get("TITLE", "xonsh"))
 
+_defaults = {"iterm.app": "iTerm2", "apple_terminal": "Terminal", "vscode": "Code", "pycharm": "PyCharm", "kate": "Kate"}
+_maps = xsh.env.get("XONTRIB_CD_TERM_PROGRAM_MAP", _defaults)
```

Comment:
this variable need to be inside the function to get the latest env if it was set later.